Project.toml

```diff
@@ -1,20 +1,22 @@
 name = "EasyFFTs"
 uuid = "08be435b-48e7-4090-a646-9e3615ae1968"
 authors = ["KronosTheLate"]
-version = "0.3.0"
+version = "0.4.0"
```

Comment:
I am rather sure that for versions 0.X.Y, incrementing X means that the change is breaking. It is only after the 1.X release that you increment the X for non-breaking changes. So this should be 3.1